examples/00-fluent/exhaust_system.py

```diff
@@ -48,6 +48,8 @@
 import ansys.fluent.core as pyfluent
 from ansys.fluent.core import examples
 
+# Do not set pyfluent.CONTAINER_MOUNT_PATH. It has set to run this example in GitHub container only.
```

Comment:
@hpohekar these statements need to be written more clearly.

Reply:
@seanpearsonuk 

We are going to make changes in CI script to remove this comment from example script. (discussed with @mkundu1)